### DISCUSSION TITLE
除了RapidUnWrap，有没有其他前处理模型或代码，提高单行识别准确率？将原图处理成无noise的高清晰白底黑字图？

### DISCUSSION THREAD

**Discussion by nissansz** [RapidAI/TableStructureRec#82] [Q&A]
除了RapidUnWrap，有没有其他前处理模型或代码，提高单行识别准确率？将原图处理成无noise的高清晰白底黑字图？

**Comment by Joker1212** [comment]
https://github.com/ZZZHANG-jx/DocRes

**Reply by nissansz** [reply]
谢谢。怎么转换成RapidUnWrap这样的项目，方便调用？


2. Appearance Enhancement
3. Deshadow
4. Dewarping
5. Deblur
6. Binarization

**Reply by Joker1212** [reply]
> 谢谢。怎么转换成RapidUnWrap这样的项目，方便调用？
> 
> 2. Appearance Enhancement
> 3. Deshadow
> 4. Dewarping
> 5. Deblur
> 6. Binarization

这个目前暂时只能你自己想办法了，就是写代码，目前我这边有其他事情很忙，未来可能会集成到RapidUnwrapped里面

**Reply by nissansz** [reply]
完全不会写。只能等你更新RapidUnwrapped

**Comment by nissansz** [comment]
检测可视化时可以看到有文本行，但是检测+识别可视化时，有的文本行消失了，这是哪个参数控制的？确保所有检测到的文本行在识别可视化也有结果？ 
![image](https://github.com/user-attachments/assets/9eba509b-3a11-4154-ac2c-f3744ed1f3f5)

**Reply by Joker1212** [reply]
这个没办法保证完全可行，使用多语言识别模型试试呢，没有参数能保证都能识别哈

**Reply by nissansz** [reply]
好像是方向识别错误导致的，单独分2步，遍历好像可以解决

**Reply by nissansz** [reply]
改成这个可以解决text_score=0.01,

**Reply by Joker1212** [reply]
ok

**Comment by nissansz** [comment]
单字符识别得分多少，才能确保正确？==1？
